sc/social/like/utils.py

```diff
@@ -112,3 +117,34 @@ def get_valid_objects(brains):
             logger.warn(msg.format(b.getPath()))
             continue
         yield obj
+
+
+def get_image_fallback(site=None):
+    from sc.social.like.interfaces import ISocialLikeSettings
+    if site is None:
+        site = getSite()
+    registry = getUtility(IRegistry)
+    settings = registry.forInterface(ISocialLikeSettings, check=False)
+    site_url = site.absolute_url()
+
+    if getattr(settings, 'image_fallback', False):
+        filename, data = b64decode_file(settings.image_fallback)
+        return '{0}/@@sociallike-image-fallback/{1}'.format(
```

Comment:
let's just return the path relative to the site root; also, don't use `format()` in this case as is pretty slow; a simple concatenation will make the work faster: `return '/@@sociallike-image-fallback/' + filename`

Reply:
@hvelarde you forgot about the `site_url` variable in your example

Reply:
no, I want to remove that call from here.

Reply:
ok